### DISCUSSION TITLE
How about sent file with anycable.

### DISCUSSION THREAD

**Discussion by thiensubs** [anycable/anycable#121] [General]
Dear all,
I write an example integrated using AnyCable, one feature I had a problem with is I try to sent a file (blob) with AnyCable.
Can somebody help me invest more in this?
My current project is here  https://github.com/thiensubs/actioncable-streamvideo-demo.

Regards,

**Comment by palkan** [comment]
Action Cable (and thus, AnyCable) only supports text messages; thus, sending binary data is not supported out-of-the-box.
The only way to do that is to build a custom `anycable-go` server with such capabilities.

What task do you want to accomplish?

**Reply by sponomarev** [reply]
I guess it's possible to wrap a file in base64 and send it as a part of a message, but as far as I remember we have a message size limit on the WS server side.

**Reply by palkan** [reply]
> as far as I remember we have a message size limit on the WS server side

True. Though it's configurable.

**Reply by thiensubs** [reply]
In my opinion, I think wrap with base64 is good, but it also sending a lot of lines to screen log or log file. It will take a lot of I/O. 
So I think I will try to kind of blob or Uint8Array because it only sent the file without too many log lines.
